packages/ui-extensions/src/surfaces/admin/components/patterns/examples/interstitialNav.html

```diff
@@ -0,0 +1,55 @@
+<s-section heading="Preferences">
+  <s-stack gap="none" border="base" borderRadius="base" overflow="hidden">
+    <s-clickable
+      padding="small-100"
+      href="/app/settings/shipping"
+      accessibilityLabel="Configure shipping methods, rates, and fulfillment options"
+    >
+      <s-grid gridTemplateColumns="1fr auto" alignItems="center" gap="base">
+        <s-box>
+          <s-heading>Shipping & fulfillment</s-heading>
+          <s-paragraph color="subdued">
+            Shipping methods, rates, zones, and fulfillment preferences.
+          </s-paragraph>
+        </s-box>
+        <s-icon type="chevron-right"></s-icon>
+      </s-grid>
+    </s-clickable>
+    <s-box paddingInline="small-100">
```

Comment:
I assume you got this from somewhere else, but the padding for the s-divider looks weird, especially when you hover over one of the s-clickables. Do we really want the padding?